src/CONST.js

```diff
@@ -330,6 +330,7 @@ const CONST = {
         NUMERIC: 'numeric',
         PHONE_PAD: 'phone-pad',
         NUMBER_PAD: 'number-pad',
+        DECIMAL_PAD: 'decimal-pad',
```

Comment:
I know in the original proposal we agreed in using the `decimal-pad` keyboard type. However, from reading [this](https://lefkowitz.me/visual-guide-to-react-native-textinput-keyboardtype-options/), it looks like `numeric` and `decimal-pad` are equivalent. So could we just use the existing `numeric` option instead of adding this new `decimal-pad` option? 

cc @parasharrajat

Reply:
Hmm... If they are the same, it does not matter unless we depend on the virtual keyboard on Android/IOS. So @thesahindia Could you retest on platforms after this check?

Reply:
Both keyboards are giving the same results. So which one should be used? @parasharrajat
<img width="1296" alt="Screenshot 2021-12-17 at 12 49 45 AM" src="https://user-images.githubusercontent.com/83179295/146434971-64581b67-3035-45a4-81e3-2824cd37a31e.png">

Reply:
>  So could we just use the existing numeric option instead of adding this new decimal-pad option?

From puneet's comment.